lib/pure/collections/lists.nim

```diff
@@ -626,6 +642,20 @@ proc add*[T](L1, L2: var DoublyLinkedList[T]) {.since: (1, 5).} =
   L2.head = nil
   L2.tail = nil
 
+proc add*[T](L1: var DoublyLinkedList[T], L2: DoublyLinkedList[T]) {.since: (1, 5).} =
```

Comment:
factor (as other procs do in lists.nim):
```nim
proc add*[T: SomeLinkedList](a: var T, b: T) {.since: (1, 5).} =
  var c = b.copy
  a.addMove c
```